docs/conf.py

```diff
@@ -20,181 +20,175 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath(".."))
```

Comment:
why change this file?

Reply:
Seems like it was caught by black, should be fine?